From 97b82bbdf275f776683efdb045d5919d007564b3 Mon Sep 17 00:00:00 2001
From: Krzysztof Witkowski <krzysztof.witkowski@grinn-global.com>
Date: Fri, 11 Jul 2025 13:52:28 +0200
Subject: [PATCH] Modify Makefile

---
 modules/Makefile | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/modules/Makefile b/modules/Makefile
index d5ea855..07c8d4f 100644
--- a/modules/Makefile
+++ b/modules/Makefile
@@ -46,7 +46,7 @@ define do_build
 	$(MAKE) -C $(KERNEL_DIR) M=$(MODULE_DIR) $(KERNEL_ARCH) $(1) CCFLAGS="$(VERSION_FLAGS)"
 endef
 
-all: check_kbuild_config
+all:
 	@echo "** RT Driver Version : $(RT_VERSION_MAJOR).$(RT_VERSION_MINOR).$(RT_VERSION_PATCH) **"
 	@echo "** PCie Driver Version : $(PCIE_VERSION_MAJOR).$(PCIE_VERSION_MINOR).$(PCIE_VERSION_PATCH) **"
 	$(call do_build,modules)
@@ -54,11 +54,13 @@ all: check_kbuild_config
 sparse:
 	$(call do_build,modules C=1 W=1)
 
-install:
+modules_install:
 	$(MAKE) -C $(KERNEL_DIR) M=$(MODULE_DIR) $(KERNEL_ARCH) modules_install
 
 clean:
-	$(MAKE) -C $(KERNEL_DIR) M=$(MODULE_DIR) $(KERNEL_ARCH) clean
+	rm -rf *.o *~ core .depend .*.cmd *.ko *.mod.c
+	rm -rf Module.markers Module.symvers modules.order
+	rm -rf .tmp_versions Modules.symvers
 
 check_kbuild_config:
 	@if [ ! -f "$(KERNEL_DIR)/include/generated/autoconf.h" -o ! -f "$(KERNEL_DIR)/include/config/auto.conf" ]; then \
-- 
2.43.0

