From 167c3f3a55be3ff931532c7412c2984eef4a5104 Mon Sep 17 00:00:00 2001
From: Mateusz Koza <mateusz.koza@grinn-global.com>
Date: Fri, 21 Feb 2025 16:11:25 +0100
Subject: [PATCH] Modify Service

---
 service/dxrt.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/service/dxrt.service b/service/dxrt.service
index 4f8169a..d79f2b5 100644
--- a/service/dxrt.service
+++ b/service/dxrt.service
@@ -2,8 +2,8 @@
 Description=DX-RT Service
 
 [Service]
-ExecStart=/usr/local/bin/dxrtd
-WorkingDirectory=/usr/local/bin/
+ExecStart=/usr/bin/dxrtd
+WorkingDirectory=/usr/bin/
 Restart=on-failure
 
 [Install]
